extensions/wikia/AuthModal/css/AuthModal.scss

```diff
@@ -12,11 +12,15 @@
 	box-sizing: border-box;
 	opacity: 1;
 	padding: 50px 10%;
+
+	&.visible {
```

Comment:
This needs to stay, as $blackout has too low z-index to cover global nav. If we unehance it there, other things will get messed, so it's the safest way to increase z-index here.